build/Uno.UI.Lottie.nuspec

```diff
@@ -0,0 +1,49 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+	<id>Uno.UI.Lottie</id>
+	<version>1.0.0-to-be-set-by-ci</version>
+	<title>Uno.UI.Lottie</title>
+	<authors>nventive</authors>
+	<owners>nventive</owners>
+	<requireLicenseAcceptance>false</requireLicenseAcceptance>
+	<projectUrl>https://github.com/nventive/Uno</projectUrl>
+	<iconUrl>https://nv-assets.azurewebsites.net/logos/uno.png</iconUrl>
+	<description>
+	  Support for Lottie on Uno.UI. This is a fork of the Windows samples app
```

Comment:
Fork of the nuget package, not the app ?

Reply:
You're right. thanks.